x-pack/plugins/cross_cluster_replication/public/app/constants/form_schemas.js

```diff
@@ -0,0 +1,137 @@
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import Joi from 'joi';
+import { INDEX_ILLEGAL_CHARACTERS_VISIBLE } from 'ui/indices';
+
+import { indexNameValidator } from '../services/input_validation';
+
+const indexNameIllegalCharacters = INDEX_ILLEGAL_CHARACTERS_VISIBLE.join(' ');
+
+/* eslint-disable max-len */
+export const follwerIndexFormSchema = {
+  name: {
+    label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.sectionFollowerIndexNameTitle', {
+      defaultMessage: 'Name'
+    }),
+    description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.sectionFollowerIndexNameDescription', {
+      defaultMessage: 'A name for the follower index.'
+    }),
+    helpText: i18n.translate('xpack.crossClusterReplication.followerIndexForm.indexNameHelpLabel', {
+      defaultMessage: 'Spaces and the characters {characterList} are not allowed.',
+      values: { characterList: <strong>{indexNameIllegalCharacters}</strong> }
+    }),
+    validator: indexNameValidator,
+  },
+  leaderIndex: {
+    label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.sectionLeaderIndexTitle', {
+      defaultMessage: 'Leader index'
+    }),
+    description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.sectionLeaderIndexDescription', {
+      defaultMessage: 'The leader index you want to replicate from the remote cluster.'
+    }),
+    helpText: i18n.translate('xpack.crossClusterReplication.followerIndexForm.indexNameHelpLabel', {
+      defaultMessage: 'Spaces and the characters {characterList} are not allowed.',
+      values: { characterList: <strong>{indexNameIllegalCharacters}</strong> }
+    }),
+    validator: indexNameValidator,
+  },
+  advanced: {
+    maxReadRequestOperationCount: {
+      label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxReadRequestOperationCountTitle', {
+        defaultMessage: 'Max read request operation count'
+      }),
+      description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxReadRequestOperationCountDescription', {
+        defaultMessage: 'The maximum number of operations to pull per read from the remote cluster.'
+      }),
+      validator: Joi.number().allow(''),
+    },
+    maxOutstandingReadRequests: {
+      label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxOutstandingReadRequestsTitle', {
+        defaultMessage: 'Max outstanding read requests'
+      }),
+      description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxOutstandingReadRequestsDescription', {
+        defaultMessage: 'The maximum number of outstanding reads requests from the remote cluster.'
```

Comment:
should this be `outstanding read requests`? (I think the ES documentation where this is copied from also has the same typo)

Reply:
The danger of copy/pasting.... yes will change